### DISCUSSION TITLE
Midi only track output and disabling track voice usage

### DISCUSSION THREAD

**Discussion by maks** [bangcorrupt/hacktribe#177] [Ideas]
I thought this had come up before but I couldn't find it in discussion or issues so apologies if its been answered already.

So the E2's 16 tracks makes its potentially really useful to sequence other midi instruments to save on the E2's own limited voices, but afaik even if a track has is volume turned completely down it will still use up a voice. 

So I wonder if it would be possible to somehow mark tracks as "midi only" so as to stop them from using up a internal voice?

**Comment by bangcorrupt** [comment]
I don't think this has been discussed before, MIDI tracks are a good idea.  There's a lot of work to do reversing how the DSP constructs audio chains, maybe I'll know more after fixing #71.  

To minimize the DSP load for a track, set Osc to Audio In, switch the filter off and set IFX to Punch.  Interestingly, the sequencer looks like it supports 8 notes per part, but has been limited to 4 to reduce the DSP load.
